charts/nexus-iq/values.yaml

```diff
@@ -28,6 +28,20 @@ iq:
   env:
     - name: JAVA_OPTS
       value: "-Djava.util.prefs.userRoot=${SONATYPE_WORK}/javaprefs"
+  # Configures the liveness probe for IQ pod
+  livenessProbe:
+    initialDelaySeconds: 10
+    periodSeconds: 10
+    failureThreshold: 3
+    timeoutSeconds: 2
+    successThreshold: 1
+  # Configures the liveness probe for IQ pod
+  readinessProbe:
+    initialDelaySeconds: 10
+    periodSeconds: 10
+    failureThreshold: 3
+    timeoutSeconds: 2
+    successThreshold: 1
```

Comment:
Does above mean those values will be the default if users do not specify?
If so, aren't above a bit too strict for larger IQ + H2?

Reply:
Hi @hajimeo, yes those will be the default values for users that do not specify them on a custom values file, and on this initial PR, I am just using the default values that Kubernetes have for those params, so we do not change the current behavior. Look at this: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes

Regarding if these values are too strict, maybe you are right, and we have here an opportunity to have more accurate values. I am going to check with the team in detail. 

Also, if you have any suggestions for those numbers, please let us know.

Reply:
Edit: I was wrong when I said I was using the default values, actually I updated the values for `initialDelaySeconds` and `initialDelaySeconds`, default values were `initialDelaySeconds:0` and `timeoutSeconds:1`

Reply:
@hajimeo do you have ideas for more appropriate values to specify here? we need to balance that time needed for the first migration startup vs daily operation. In daily operation, the server failing for many minutes before being restarted could be too long, but I guess that's "liveness" not "readiness".
To facilitate the long migration time, we _may_ want to drastically extend readiness times and thresholds. Is there a downside to waiting too long for the startup? probably not.

Reply:
I just updated the `readinessProbe` default values to try to cover a larger IQ setup. I think with those values we should cover a lot of scenarios. In any case, if needed, now users can override those values with a custom values file